src/specs/02_advanced/qe81_network_upgrade.spec

```diff
@@ -323,3 +324,58 @@ PartyProtection
 * "rawContract2"'s `get()` function execution in "Node1" returns "25"
 * "rawContract2"'s `get()` function execution in "Node4" returns "25"
 * "rawContract2"'s `get()` function execution in "Node3" returns "0"
+
+ Upgrade node 3
+* Stop and start "Node3" using quorum version <q_upgd_version> and tessera version <t_upgd_version>
+ Wait for "quorum" state in "Node3" to be "up"
```

Comment:
shouldn't this be a step 
`* Wait for "quorum" state in "Node3" to be "up"`
same applied for the rest

Reply:
Ok I commented it out because I think it was not working, I will test again. Thanks